itou/approvals/admin.py

```diff
@@ -22,15 +23,35 @@ class JobApplicationInline(admin.StackedInline):
         "approval_number_sent_at",
         "approval_delivery_mode",
         "approval_manually_delivered_by",
+        "employee_record_status",
     )
-    raw_id_fields = ("job_seeker", "to_siae", "approval_manually_delivered_by")
+    raw_id_fields = (
+        "job_seeker",
+        "to_siae",
+        "approval_manually_delivered_by",
+    )
+
+    # Mandatory for "custom" inline fields
+    readonly_fields = ("employee_record_status",)
 
     def has_change_permission(self, request, obj=None):
         return False
 
     def has_add_permission(self, request, obj=None):
         return False
 
+    # Custom read-only fields as workaround :
+    # there is no direct relation between approvals and employee records
+    # (YET...)
+    @admin.display(description="Statut de la fiche salarié")
+    def employee_record_status(self, obj):
+        if employee_record := obj.employee_record.first():
+            url = f"/admin/employee_record/employeerecord/{employee_record.id}"
```

Comment:
on peut éviter cette URL hard-codée avec un `reverse`… avec une autre chaine hardcodée. C’est pas forcément mieux, mais c’est peut-être plus cohérent avec le reste de l’appli:
```suggestion
            url = reverse('admin:employee_record_employeerecord_change', args=[employee_record.id])
```

Reply:
Oui, j'avais la flemme, et c'est plus propre. Merci !